piezo_web_app/PiezoWebApp/tests/integration_tests/submit_job_test.py

```diff
@@ -337,7 +339,8 @@ def test_all_optional_inputs_defined_to_maximum_succeeds(self):
             'status': 'success',
             'data': {
                 'message': 'Job driver created successfully',
-                'job_name': 'test-python-job-abcd1'
+                'job_name': 'test-python-job-abcd1',
+                'spark_ui': 'http://0.0.0.0:1/proxy:test-python-job-abcd1-ui-svc:4040'
```

Comment:
Is there an integration test for a `submitjob` request where the Spark UI fails to initialise?

Reply:
0d30a4e